contrib/dnsmasq/patches/conf.patch

```diff
@@ -0,0 +1,33 @@
+diff --git a/dnsmasq.conf.example b/dnsmasq.conf.example
+index f1fb2d1..1851f0b 100644
+--- a/dnsmasq.conf.example
++++ b/dnsmasq.conf.example
+@@ -22,7 +22,7 @@
+ 
+ # Uncomment these to enable DNSSEC validation and caching:
+ # (Requires dnsmasq to be built with DNSSEC option.)
+-#conf-file=%%PREFIX%%/share/dnsmasq/trust-anchors.conf
++#conf-file=/usr/share/dnsmasq/trust-anchors.conf
+ #dnssec
+ 
+ # Replies which are not DNSSEC signed may be legitimate, because the domain
+@@ -106,8 +106,8 @@
+ 
+ # If you want dnsmasq to change uid and gid to something other
+ # than the default, edit the following lines.
+-#user=
+-#group=
++user=_dnsmasq
++group=_dnsmasq
```

Comment:
does it work by default by using these (since i meant to leave it commented?)

if it binds e.g. :25 before dropping privs it should be fine, in which case keep it- do check though

Reply:
seems to work fine, I could also just patch the config.h with these groups like alpine?

Reply:
good idea to do that too yeah

Reply:
if it works fine you should leave it uncommented

Reply:
actually yea it uses the patch default so nvm